public/app/features/plugins/admin/components/SearchField.tsx

```diff
@@ -12,6 +13,8 @@ export const SearchField = ({ value, onSearch }: Props) => {
   const [query, setQuery] = useState(value);
   const styles = useStyles2(getStyles);
 
+  useDebounce(() => onSearch(query ?? ''), 600, [query]);
```

Comment:
nit: this is really subjective but `600` felt a bit too long to wait whilst I was testing this change. 🤔

Reply:
ah, ok ok - I was playing around with different values and was trying to find the highest one that felt okay. But I can lower it a bit. Great feedback!